hydromt_wflow/workflows/glaciers.py

```diff
@@ -141,8 +141,8 @@ def convStoremm(areakm2, volm3, rhoice=0.9):
 
 def glacierattrs(
     gdf,
-    TT=1.3,
-    Cfmax=5.3,
+    snow_tt=1.3,
+    snow_degree_day_coefficient=5.3,
```

Comment:
You can decide to undo renaming of the function argument but if you keep the renaming, this is actually glacier_ttm, glacier_cfmax etc. If you rename, please rename all including SIfrac (glacier_snow_to_ice_fraction is the new default staticmap name)